### PR TITLE
Update AGRC Director Info

### PR DESCRIPTION
**Pull request by stdavis**
Supercedes #788 because I'm not smart enough to add to that one. :)

Tweaks the emphasis and adds area codes to satisfy my OCD.

### CODE REVIEW

File: about/contact/index.html [hunk 1]
```diff
@@ -40,50 +40,50 @@ <h4>AGRC Staff Information</h4>
       <td colspan="4" style="padding:0;height:1px;border:0;background-color:#e67e15"></td>
     </tr>
     <tr>
-      <th class="text-left" colspan="4"><strong>Director</strong></th>
```
Comment (steveoh) [inline about/contact/index.html]: Shouldn't this continue to say director?

Comment (stdavis) [reply about/contact/index.html]: I think that "AGRC Team Lead" @brigsz's preferred title over "Director".

Comment (steveoh) [reply about/contact/index.html]: take me to your leader

Comment (stdavis) [reply about/contact/index.html]: 👽

**Comment by brigsz** [comment]
Ha! I am unsure, Director seems a bit heavy

On Feb 23, 2018, at 4:22 PM, Scott Davis <notifications@github.com> wrote:

*@stdavis* commented on this pull request.
------------------------------

In about/contact/index.html
<https://github.com/agrc/gis.utah.gov/pull/789#discussion_r170392498>:

> @@ -40,50 +40,50 @@ <h4>AGRC Staff Information</h4>
       <td colspan="4"
style="padding:0;height:1px;border:0;background-color:#e67e15"></td>
     </tr>
     <tr>
-      <th class="text-left" colspan="4"><strong>Director</strong></th>

I think that "AGRC Team Lead" @brigsz <https://github.com/brigsz>'s
preferred title over "Director".

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/agrc/gis.utah.gov/pull/789#discussion_r170392498>, or mute
the thread

**Pull request merged**